### PR TITLE
Added new active scanner TestUserAgent to alerts.xml

### PR DESCRIPTION
**Pull request by denniskniep**
Implemented a new active scanner. This scanner should be added to the alerts.xml. https://github.com/zaproxy/zap-extensions/pull/47

### CODE REVIEW

File: src/doc/alerts.xml [hunk 1]
```diff
@@ -86,7 +86,8 @@ This list the alerts
 10046	Insecure Component
 10047	HTTPS Content Available via HTTP
 10048	ShellShock
-10049  Image Location Scanner - passive scanner finding
+10049   Image Location Scanner - passive scanner finding
```
Comment (thc202) [inline src/doc/alerts.xml]: Indentation changes are better if done in other commit than the actual changes (but I guess a little change like this is just fine). [1]

[1] https://github.com/zaproxy/zaproxy/wiki/DevGuidelines

Comment (kingthorin) [inline src/doc/alerts.xml]: Not sure if it matters at all but over here you mentioned intent to use 10050: https://github.com/zaproxy/zap-extensions/pull/47#issuecomment-146001555

Comment (denniskniep) [inline src/doc/alerts.xml]: Yes, the id 10050 is in line 90. Additionally I removed a redundant space at line 89.

**Comment by kingthorin** [comment]
Np, only saw the change the commented on.

File: src/doc/alerts.xml (hunk 1 above)
Comment (kingthorin) [inline src/doc/alerts.xml]: Np, only saw the change the commented on.

**Pull request merged**